experiments/azure/cyclecloud/cpu/experiment/README.md

```diff
@@ -87,125 +87,143 @@ rm singularity-ce*.tar.gz
 
 Batch script:
 
-```console
+```bash
+size=128
 oras login ghcr.io --username vsoch
-srun --time=00:04 -N 2 slurm-single-node-benchmarks.sh
+srun --time=00:04 -N ${size} slurm-single-node-benchmarks.sh
 rm -rf test_file*
```

Comment:
I just want to double check you are using the `-tmpfile` prefix image? If so, this shouldn't be generated in the PWD directory anymore - it (along with the machine.xml|svg) are generated and cleaned up in an isolated temporary directory.

Reply:
Yes, there aren't any tmpfiles with the `tmpfile` image that I've been using.

Reply:
Gotcha. We could remove the `rm -rf` then, if desired.